org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/BundleRenderer.java

```diff
@@ -198,7 +198,7 @@ private void renderResponse(XhtmlNode root, BundleEntryResponseComponent respons
   }
 
   private void renderRequest(XhtmlNode root, BundleEntryRequestComponent request) {
-    root.para().addText("Response:");
+    root.para().addText("Request:");

```

Comment:
Should request and response be put in two variables or is this the only spot they're used?

Reply:
Only spot. There's a wider conversation to be had around i18n for the rendering but not as part of this PR